### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by szepeviktor**
🚀 https://github.com/szepeviktor/sprucecss-site/blob/patch-1/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,27 +9,32 @@
     <img src="./.github/spruce-logo-light.svg" alt="Spruce CSS" width="140">
     <br/>
   </a>
-</p>
+</h1>
```
Comment (szepeviktor) [inline README.md:12]: H1 was missing from README.

File: README.md [hunk 2]
```diff
@@ -9,27 +9,32 @@
...
 
 **Spruce CSS uses [Gatsby](https://www.gatsbyjs.org/) for its documentation.**
 
 [![Netlify Status](https://api.netlify.com/api/v1/badges/32f689b9-2e24-462f-be66-22e1233d7f9b/deploy-status)](https://app.netlify.com/sites/sprucecss/deploys)
 
-## Installation
+## Contributing
```
Comment (szepeviktor) [inline README.md:18]: More precise naming.

File: README.md [hunk 3]
```diff
@@ -9,27 +9,32 @@
...
 
 1.  **Clone the repository**
 
+    ```shell
+    git clone https://github.com/conedevelopment/sprucecss-site.git
+    ```
```
Comment (szepeviktor) [inline README.md:24]: Add cloning instruction.

**Comment by adamlaki** [comment]
Thanks Viktor!

**Pull request merged**